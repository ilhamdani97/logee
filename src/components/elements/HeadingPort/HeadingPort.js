import React from 'react';
import PropTypes from 'prop-types';
import styles from './styles.scoped.css';
import { Typography } from 'logeeport-ui';

const HeadingPort = ({ title }) => {
  return (
    <div className={styles.root}>
      <Typography variant="h4">{title}</Typography>
    </div>
  );
};

HeadingPort.defaultProps = {
  title:'',
};

HeadingPort.propTypes = {
  title:PropTypes.string,
};

export default HeadingPort;
