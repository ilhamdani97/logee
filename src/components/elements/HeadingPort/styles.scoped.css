.root {
  position: relative;
  padding-top: 20px;
  padding-bottom: 20px;
  padding-right: 20px;
  border-bottom: 2px solid var(--grey-color-3);
  text-transform: capitalize;
}

.root h4{
  color: #000000;
  font-weight: 600;
}