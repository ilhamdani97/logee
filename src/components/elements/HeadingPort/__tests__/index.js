import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import HeadingPort from '../HeadingPort';

describe('src/components/elements/HeadingPort', () => {
  test('render', () => {
    const shallow = new ShallowRenderer();
    const tree = shallow.render(<HeadingPort/>);
    expect(tree).toMatchSnapshot();
  });
});
