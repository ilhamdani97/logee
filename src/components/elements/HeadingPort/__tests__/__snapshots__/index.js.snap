// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/elements/HeadingPort render 1`] = `
<div>
  <Typography
    spacing=""
    variant="h4"
  >
    
  </Typography>
</div>
`;
